autocrop/autocrop.py

```diff
@@ -226,8 +276,13 @@ def main(input_d,
             if input_filename != output_filename:
                 # Move the file to the output directory
                 shutil.move(input_filename, output_filename)
-            # Write the cropped image
-            cv2.imwrite(output_filename, image)
+            # Encode the image as an in-memory PNG
+            img_png = cv2.imencode(".png", image)[1].tostring()
+            # Read the PNG data
+            img_new = Image.open(io.BytesIO(img_png))
```

Comment:
### Important
The reason [the Windows tests fail](https://ci.appveyor.com/project/leblancfg/autocrop/builds/21742939/job/pgn8q8ah170dmax7#L193) is because the files aren't closed after reading them in. The Python idiom to read (and close) files is the following, called the context manager:

~~~python
with open(path) as file:
    img = Image(file)
~~~

As suggested previously, no need to decode to PNG, but please **do** use the context manager to read in the file with PIL. I strongly believe it should make the tests pass as they currently stand :smile: